setup.cfg

```diff
@@ -49,6 +49,7 @@ install_requires =
     dask[array]>=2.15.0,!=2.28.0  # https://github.com/napari/napari/issues/1656
     imageio>=2.20,!=2.22.1
     jsonschema>=3.2.0
+    lazy_loader==0.2
```

Comment:
@stefanv I don't like hard pinning here. If, say, `lazy_loader` updates to 0.3 and then scipy requires 0.3+, then napari will be considered incompatible with scipy even though that might not be the case at all. [This long post](https://iscinumpy.dev/post/bound-version-constraints/) convinced me that preemptive pinning is a bad idea in most cases, because users can get around a failing update by downgrading, they have no recourse when the dependency pin is incorrect and in fact everything would work fine without it.

If you think that lazy_loader is still evolving too quickly *not* to pin, then I think that implies we are not actually ready to unvendor yet...